homeassistant/components/wemo.py

```diff
@@ -91,6 +92,7 @@ def discovery_dispatch(service, discovery_info):
         if device is None:
             device = pywemo.discovery.device_from_description(url, None)
 
-        discovery_info = (device.name, device.model_name, url, device.mac)
+        discovery_info = (device.name, device.model_name, url, device.mac,
```

Comment:
I found device.name somewhat useful when debugging things, this is the friendly name that a user has set in the wemo app.